src/pages/landing/deriv-go/components/_tabs.tsx

```diff
@@ -144,7 +174,7 @@ TabPanel.propTypes = {
     className: PropTypes.string,
 }
 
-const Tabs = ({ children, is_reverse, className, max_width }) => {
+const Tabs = ({ children, is_reverse, className, max_width }: TabsTYpe) => {
```

Comment:
```suggestion
const Tabs = ({ children, is_reverse, className, max_width }: TabsProps) => {
```
Should be Props for components.